ui/js/controllers/perf/graphs.js

```diff
@@ -649,7 +666,17 @@ perf.controller('GraphsCtrl', [
                     $scope.seriesList = [];
                     addSeriesList([]);
                 }
-
+                if ($stateParams.tooltip) {
```

Comment:
This and the variables below should change tooltip -> selected. As it is, this change breaks the feature.

Please test your changes before submitting a new PR, it saves a lot of time on both sides.